ramls/jsonSchemas.raml

```diff
@@ -0,0 +1,38 @@
+#%RAML 1.0
+
+title: JSON Schema API
+version: v1
+baseUri: http://localhost:8081/{version}
+
+documentation:
+  - title: JSON Schema API
+    content: This documents the API calls that can be made to get JSON Schemas of a module defined by X-Okapi-Module-Id header
+
+types:
+  errors: !include ../schemas/errors.schema
+
+/_/jsonSchemas:
+  get:
+    description: Get JSON Schema names for the given module defined by header X-Okapi-Module-Id
```

Comment:
Explain the two cases in the description: List all schema names, get a schema for a specific path.
Can you add an example for the list of schema names including a schema in a subdirectory?